sourcetracker/_sourcetracker.py

```diff
@@ -462,6 +479,12 @@ def gibbs_sampler(sink, cp, restarts, draws_per_restart, burnin, delay):
         additional samples will be drawn every `delay` number of passes. This
         is also known as 'thinning'. Thinning helps reduce the impact of
         correlation between adjacent states of the Markov chain.
+    autoc : function or None
+        Determines what mode the function will be run in. If not `None`,
+        `autoc` will be used to determine if the chain has converged. `autoc`
+        must accept two arguments; the `final_envcounts` array and an index
```

Comment:
Add: *`autoc` should return a bool indicating if convergence has been achieved (`True`) or not (`False`).*